django_ninja_crudl/service.py

```
@@ -1,362 +1,362 @@
+#         related_models: list[str] = []
+#         many_to_many_models: list[str] = []
+#         related_fields: list[str] = []
+#         property_fields: list[str] = []
+#
+#         for field_name in self.config.list_schema.model_fields:
+#             attr = getattr(self.model_class, field_name, None)
+#
+#             # Skip @property methods here
+#             if attr and isinstance(attr, property):
+#                 property_fields.append(field_name)
+#                 # TODO: Implement a way to avoid N+1 queries when using @property methods
```

Ruff (0.8.2)

Missing author in TODO; try: `# TODO(<author_name>): ...` or `# TODO @<author_name>: ...`

(TD002)